View/Screens/RoleAssignmentScreen.xaml.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Controller.DbControllers;
+using Model.Database.Contexts;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using View.DataContexts;
+using View.ListItems;
+
+namespace View.Screens
+{
+    /// <summary>
+    /// Interaction logic for RoleAssignmentScreen.xaml
+    /// </summary>
+    public partial class RoleAssignmentScreen : ResourceDictionary
+    {
+        private static RoleAssignmentDataContext instance;
+        public static RoleAssignmentDataContext Instance
+        {
+            get
+            {
+                if (instance == null)
+                    instance = new RoleAssignmentDataContext();
+                return instance;
+            }
+        }
+        public RoleAssignmentScreen()
+        {
+            InitializeComponent();
+        }
+
+        public void GetUsers(object sender, KeyEventArgs e)
+        {
+            if (e.Key == Key.Return)
+            {
+                var textBox = (TextBox)sender;
+                var text = textBox.Text;
+                var Users = UserController.Create(DatabaseContext.Instance).GetList()
```

Comment:
Ditzelfde doe je toch ook al in de datacontext van dit scherm?

Reply:
Het wordt nu opgeslagen in dezelfde lijst als in de datacontext.